libs/utils/android/system.py

```diff
@@ -280,9 +280,14 @@ def gfxinfo_get(target, apk_name, out_file):
 
         :param out_file: output file name
         :type out_file: str
+
+        :param framestats: collect framestats and *append* to out_file
+        :type framestats: bool
         """
-        adb_command(target.adb_name,
-                    GET_FRAMESTATS_CMD.format(apk_name, out_file))
+        cmd = GET_FRAMESTATS_CMD.format(apk_name, out_file)
+        if framestats:
+            cmd = cmd.replace('>', 'framestats >>')
```

Comment:
I don't like this use of `replace`, there's an invisible dependency where if you change `GET_FRAMESTATS_CMD` you break this code. Maybe instead have a function like `get_framestats_cmd(x)` (with a better name than `x`, not sure what the terminology should be though), that would be used like `get_framestats_cmd(['gfxinfo', 'framstats'])` here?